app/uk/gov/hmrc/fileupload/s3/S3Service.scala

```diff
@@ -73,6 +77,8 @@ object S3Service {
   type UploadToQuarantine = (String, InputStream, Int) => Future[UploadResult]
 
   type DownloadFromBucket = (S3KeyName) => Option[StreamWithMetadata]
+
+  type DeleteFileFromQuarantineBucket = (String) => Unit
```

Comment:
Shouldn't it have the type S3KeyName ?

Reply:
As discussed IRL, will leave this as string as the value class isn't widely used in the codebase.